test/extended/olm/olmv1.go

```diff
@@ -20,66 +20,50 @@ import (
 
 const (
 	olmv1GroupName                       = "olm.operatorframework.io"
-	typeIncompatibelOperatorsUpgradeable = "InstalledOLMOperatorsUpgradeable"
+	typeIncompatibleOperatorsUpgradeable = "InstalledOLMOperatorsUpgradeable"
 )
 
-var _ = g.Describe("[sig-olmv1] OLMv1 CRDs", func() {
+var _ = g.Describe("[sig-olmv1][OCPFeatureGate:NewOLM] OLMv1 CRDs", func() {
 	defer g.GinkgoRecover()
 	oc := exutil.NewCLIWithoutNamespace("default")
 
 	g.It("should be installed", func(ctx g.SpecContext) {
-		g.Skip("Test is temporarily disabled while we make anticipated, breaking changes.")
-		// Check for tech preview, if this is not tech preview, bail
-		if !exutil.IsTechPreviewNoUpgrade(ctx, oc.AdminConfigClient()) {
-			g.Skip("Test only runs in tech-preview")
-		}
+		checkFeatureCapability(ctx, oc)
 
-		// supports multiple versions during transision
 		providedAPIs := []struct {
 			group   string
-			version []string
+			version string
 			plural  string
 		}{
 			{
 				group:   olmv1GroupName,
-				version: []string{"v1alpha1", "v1"},
+				version: "v1",
 				plural:  "clusterextensions",
 			},
 			{
 				group:   olmv1GroupName,
-				version: []string{"v1alpha1", "v1"},
+				version: "v1",
```

Comment:
I would like have a slice of strings, so that in future when we have v2 we do not have to change it again.